tinyblog/articles/2016-04-11-coc-transparency-report.md

```diff
@@ -0,0 +1,37 @@
+
+We strongly believe that [Codes of Conduct](https://2016.djangocon.eu/coc/) (CoC) are an essential part of a conference and we’ve made a lot of effort to make ours very visible:
+
+---
+
+- It was linked from all the pages of our website (header and footer);
+- Reading and agreeing with the Code of Conduct was necessary to register a conference ticket;
+- The summary of it was posted on the walls of the venue;
+- We had two emergency phone numbers (man & woman) listed on the website as well as on posters around the venue;
+- We mentioned our Code of Conduct during the conference introduction and in email communication;
+- We reviewed speakers slides to avoid potential Code of Conduct violations;
+- During the conference, we introduced a special email address (conduct@djangocon.eu) to allow people to report incidents easier. However, it was our mistake to not have it on the Code of Conduct page from the beginning, and we only introduced it halfway through the conference.
+
+We also believe it’s important for us to share the outcome of these policies and what they mean in practice.
+So in the spirit of transparency, here is an anonymised list of the CoC-related incidents that we acted upon during the conference:
+
+- An attendee was asked to remove a tweet publicly making fun of another attendee.
+- The official conference Twitter account also favourited this tweet. The favourite was removed, and a public apology was issued by conference organizers during opening remarks of the second day.
+- A conference talk contained an inappropriate slide which wasn’t caught by the conference slides review process. The speaker was asked to remove the slide from the published slidedeck. The conference organizers made sure this slide was removed from the video published after the conference. A public apology for not being thorough enough in the slides review process was issued by conference organizers during opening remarks of the second day.
+- During the conference speakers dinner, two of attendees made an inappropriate joke towards another attendee. The two attendees were reminded that not all jokes are appropriate during official conference events.
+- Person identified as not attending the conference posted an inappropriate tweet mentioning the conference official account. The person was politely asked to remove that tweet, which resulted in a series of offending messages and physical threats sent to a member of Code of Conduct team by that person. The Code of Conduct team decided to raise the issue with the DSF Code of Conduct committee, asking them to ban this person from attending future Django events and send them an email informing of actions taken.
+- One of the lightning talks contained unfortunate phrasing directed towards one of the attendees. Video from that lightning talk won’t be published online.
+We are not publishing this list to shame anyone or pat ourselves on the back.
```

Comment:
Add new line before "We are not", otherwise it just merges with last bullet point